site/en/docs/extensions/reference/offscreen/index.md

```diff
@@ -48,19 +48,36 @@ chrome.offscreen.createDocument({
 chrome.offscreen.closeDocument()
 ```
 
-The following example shows how to ensure that the offscreen document has already been created. The `setupOffscreenDocument()` function calls [`clients.matchAll()`](https://developer.mozilla.org/docs/Web/API/Clients/matchAll) to find the existing offscreen document or creates it if it doesn't already exist. Note that an extension can only have one offscreen document.
+The following example shows how to ensure that the offscreen document has already been created. The `setupOffscreenDocument()` function calls [`runtime.getContexts()`][runtime-get-contexts] to find the existing offscreen document or creates it if it doesn't already exist. Note that an extension can only have one offscreen document.
```

Comment:
As soon as you have multiple examples, you should have subheadings explaining what they are. That makes it easy to scan. Otherwise I need to read the whole of the examples section when I'm only interested in `getContexts()`.

Reply:
Actually, it is 2 examples. A basic open and close and checking if one already exists. Now that I'm writing that, shouldn't the basic example include the checking? I think some consolidation is in order here, including removing the example that doesn't include the checking.

Reply:
You're right, now I got it. The problem is not that it's two samples, but that we're mixing usage and example. I've added a usage section (merged the reasons section into it. PTAL